deploy/charts/openebs-monitoring/dashboards/Control plane/openebs-main-dashboard.json

```diff
@@ -0,0 +1,1726 @@
+{
```

Comment:
n_: directory should be : `dashboards/cstor/cstor-overview-dashboard.json`

Reply:
Fixed